catalog/spatial/registry/registry-api-impl/pom.xml

```diff
@@ -241,22 +246,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.79</minimum>
+                                            <minimum>0.68</minimum>
                                         </limit>
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.54</minimum>
+                                            <minimum>0.32</minimum>
```

Comment:
That's a pretty big drop. Check jacoco reports or run tests with coverage and see what branches have been left untested.